modules/programs/fzf.nix

```diff
@@ -88,6 +88,23 @@ in {
       '';
     };
 
+    tmux = {
+      enableShellIntegration = mkEnableOption ''
+        sets FZF_TMUX=1 which causes shell integration to use fzf-tmux
```

Comment:
This expands to `Whether to enable sets FZF_TMUX=1`.
I recommend the initial `setting <literal>FZF_TMUX=1</literal>`.